tests/dummy/app/index.html

```diff
@@ -3,23 +3,23 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>ember-toastr</title>
+    <title>Dummy</title>
```

Comment:
Title should probably be kept.

Reply:
Ah, I knew I forgot something ;) I fixed this and pushed the change!